src/stlink-lib/reg.h

```diff
@@ -2,19 +2,60 @@
 #define STLINK_REG_H_
 
 #define STLINK_REG_CM3_CPUID           0xE000ED00
+
 #define STLINK_REG_CM3_FP_CTRL         0xE0002000
-#define STLINK_REG_CM3_FP_COMP0        0xE0002008
+#define STLINK_REG_CM3_FP_COMPn(n)    (0xE0002008 + n*4)
+#define STLINK_REG_CM7_FP_LAR          0xE0000FB0
+#define STLINK_REG_CM7_FP_LAR_KEY           0xC5ACCE55
```

Comment:
Please correct the indentation in this file to match the current formatting.
Don't use tabspacing as they vary between architectures.
(I believe this is the reason for the misalignment.)

Reply:
An additional shift is deliberately used here. There are no tab chars.
STLINK_REG_CM7_FP_LAR is address of register.
STLINK_REG_CM7_FP_LAR_KEY is value for LAR register.